boards/arduino/portenta_h7/arduino_portenta_h7_stm32h747xx_m7_defconfig

```diff
@@ -25,3 +25,9 @@ CONFIG_UART_LINE_CTRL=y
 # Enable regulator
 CONFIG_REGULATOR=y
 CONFIG_REGULATOR_FIXED=y
+
+# Enable charger
+CONFIG_CHARGER=y
+
+# Enable USB Stack
+CONFIG_USB_DEVICE_STACK=y
```

Comment:
boards need to enable the minimum, e.g. to run hello world, is charger/usb stack needed there? If not, drop